app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardCSVStep/CSVDropzoneField.js

```diff
@@ -18,52 +20,69 @@ let dropzoneRef;
 class CSVDropzoneField extends React.Component {
   constructor() {
     super();
-    bindMethods(this, ['onFileDrop', 'confirmOverwrite']);
+    this.state = { unparsedFile: null, showConfirmOverwrite: false };
+    bindMethods(this, [
+      'onFileDrop',
+      'onCancelOverwrite',
+      'onConfirmOverwrite',
+      'handleUnparsedFile'
+    ]);
   }
 
   onFileDrop(acceptedFiles, rejectedFiles) {
-    const { input, columnNames } = this.props;
-    this.confirmOverwrite(() => {
-      if (acceptedFiles && acceptedFiles.length > 0) {
-        const reader = new FileReader();
-        reader.onload = () => {
-          csv.parse(reader.result, (err, csvRows) => {
-            const rowObjects = csvRows.map(row =>
-              columnNames.reduce(
-                (rowObject, key, index) => ({
-                  ...rowObject,
-                  [key]: row[index]
-                }),
-                {}
-              )
-            );
-            input.onChange(rowObjects);
-          });
-        };
-        reader.readAsBinaryString(acceptedFiles[0]);
+    const { input: { value } } = this.props;
+    if (acceptedFiles && acceptedFiles.length > 0) {
+      // Don't bother with the confirm dialog if there is no existing value to overwrite.
+      if (!value || value.length === 0) {
```

Comment:
Probably not necessary to change, but the form's value is initialized to an empty array [here](https://github.com/priley86/miq_v2v_ui_plugin/blob/migration-poc/app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardCSVStep/PlanWizardCSVStep.js#L22), so `!value` will always return true

Reply:
ah, ok.